packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js

```diff
@@ -9,61 +9,106 @@
 
 'use strict';
 
-var SyntheticWheelEvent;
-
 describe('SyntheticWheelEvent', () => {
-  var createEvent;
+  var React;
+  var ReactDOM;
+  var createMouseEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticWheelEvent = require('../SyntheticWheelEvent').default;
-
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticWheelEvent.getPooled({}, '', nativeEvent, target);
+    React = require('react');
+    ReactDOM = require('react-dom');
+    createMouseEvent = (type = '', options = {}) => {
+      const event = document.createEvent('MouseEvent');
+      event.initEvent(type, true, false);
+      return Object.assign(event, options);
     };
   });
 
   it('should normalize properties from the Event interface', () => {
-    var target = document.createElement('div');
-    var syntheticEvent = createEvent({srcElement: target});
+    const container = document.createElement('div');
+    const event = createMouseEvent('', {srcElement: container});
```

Comment:
I think this doesn't test that `srcElement` is used when `event.target` is not supported. The code just falls back to `event.target` because it exists. You need to do extra work to actually "break" `event.target` and cause the polyfill path to execute.